main/helpers.py

```diff
@@ -0,0 +1,42 @@
+from .models import Player, Field, Match
+
+
+def get_invalid_matches():
+    matches = Match.objects.all()
+    invalid_matches = []
+    for match in matches:
+        if match.player1_score != 11 and match.player2_score != 11:
+            invalid_matches.append(match)
+            continue
+
+        if match.player1_score == 11 and match.player2_score == 11:
+            invalid_matches.append(match)
+            continue
+
+        if match.player1_score > 11 or match.player2_score > 11:
+            invalid_matches.append(match)
+            continue
+
+        if match.player1_score < 0 or match.player2_score < 0:
+            invalid_matches.append(match)
+            continue
+
+    return invalid_matches
```

Comment:
This works, but if you don't mind, I'd suggest writing it in a more compact way.
It's actually easier to write the matching case - that is when a match is valid - and the negate it in the end.
First of all, the first two conditions can be merged into one that says: "if I build a list of the two scores, exactly one item of that list should be 11". You can use the `.count(item)` function if Python lists for that.
Then the second two conditions can also be merged: `0 <= foo <= 11 and 0 <= bar <= 11`.
And then you don't even have to create an empty list, and iterate and append and continue. You can accomplish everything in a short and elegant list comprehension :)
I'd actually suggest creating one more helper, `is_match_valid(match)` that does the checks I've described above, and then use that (negated) in a list comprehension in this function.